run.go

```diff
@@ -153,6 +156,17 @@ func runAction(clicontext *cli.Context) error {
 		return errors.Errorf("unknown network %q", netstr)
 	}
 
+	seccompProfile := clicontext.String("seccomp")
+	if seccompProfile != "" {
+		if seccompProfile != "unconfined" {
+			opts = append(opts, seccomp.WithProfile(seccompProfile))
+		} else {
+			// If passing "unconfined" to run a container without the default seccomp profile.
+		}
+	} else {
+		opts = append(opts, seccomp.WithDefaultProfile())
+	}
+
```

Comment:
This is how it works:

```console
# containerd's seccomp defaultProfile
$ sudo nerdctl run -it --seccomp="" alpine sh
/ # unshare
unshare: unshare(0x0): Operation not permitted

# no seccomp
$ sudo nerdctl run -it --seccomp="unconfined" alpine sh
/ # unshare --user
HOST:~$

# Enable custom seccomp
$ sudo nerdctl run -it --seccomp=_output/default.json alpine sh
/ # unshare
HOST:~$
```
refs: https://github.com/moby/moby/blob/f7bcb02f677257a5623dcb561f003e96d5f3ea3f/profiles/seccomp/default.json